Makefile

```diff
@@ -3,6 +3,9 @@
 COVERAGE          ?= 0
 DOCKER_EXTRA_ARGS ?=
 
+# TODO: fix unmaintained failing tests
+GTEST_FILTER      ?= -VEN2bTestSet.*:CurlHTTPTest.*
```

Comment:
I moved default gtest_filter value to the Makefile since it is quite unusual to have it in the google test main.cpp.